game/menu/gamemanager.gd

```diff
@@ -13,22 +20,162 @@ func _ready():
 
 
 func load_game(game_cfg: ConfigFile):
+	_last_loaded_game = game_cfg.get_meta("folder_name")
+	_started_playing_game = OS.get_ticks_msec()
 	# load the games main scene
-	var scene = load(game_cfg.get_meta("folder_path") + game_cfg.get_value("game", "main_scene"))
-
+	var scene = load(
+		(
+			"res://games/%s/%s"
+			% [
+				game_cfg.get_meta("folder_name"),
+				game_cfg.get_value("game", "main_scene"),
+			]
+		)
+	)
 	var err := get_tree().change_scene_to(scene)
 	if err != OK:
 		prints("Error", err)
 		return
 	_main.hide()
 
 
+## Save data to a file with for the given game.
+## This is a private function, don't use this in a game
+func _save_data(file_name: String, data, game: String) -> int:
```

Comment:
If it is the game_id please call it game_id and not game.
And you should write down somewhere that the game_id is the same as the game folders name.
```suggestion
func _save_data(file_name: String, data, game_id: String) -> int:
```

Reply:
Now that it is merged in `res://menu/README.md`